client/src/store/News.js

```diff
@@ -0,0 +1,57 @@
+
+import Parser from 'rss-parser';
+
+const SET_FEED = "jackpotDegen/news/SET_FEED";
+const SET_IMGS = "jackpotDegen/news/SET_IMG"
+
+export const setFeed = (value) => {
+  return {
+    type: SET_FEED,
+    feed: value,
+  }
+}
+
+export const setImg = (value) => {
+  return {
+    type: SET_IMGS,
+    imgs: value,
+  }
+}
+export const getFeed = () => async dispatch => {
+    let parser = new Parser();
+    const CORS_PROXY = "https://cors-anywhere.herokuapp.com/"
```

Comment:
Word of warning that your app can crash if you use this (f the `cors-anywhere` site is down)! You should refactor to make this fetch request from the backend (with [node-fetch](https://github.com/node-fetch/node-fetch)), so that your frontend sends a fetch request to your backend API with the proxy, like how you did for the other thunk action creators!